src/library/scala/collection/Iterable.scala

```diff
@@ -259,7 +259,7 @@ trait IterableOps[+A, +CC[_], +C] extends Any with IterableOnce[A] with Iterable
     *        x >  0       if this.size >  otherSize
     *   }}}
     *  The method as implemented here does not call `size` directly; its running time
-    *  is `O(size min _size)` instead of `O(size)`. The method should be overwritten
+    *  is `O(size min otherSize)` instead of `O(size)`. The method should be overridden
```

Comment:
Given the existence of `knownSize`, it's not actually super important to override this method anymore (as long as `knownSize` is overridden). Should we remove this part of the scaladoc?

Reply:
I think maybe add this `knownSize` part to it.